### ISSUE TITLE
add departure track to the list

### ISSUE THREAD

**Issue by unclesam87** [raywo/MMM-PublicTransportHafas#68]
Hello,

my magicmirror is up and running, and finally i can also see my trains, but without knowing from which track they departure it is hard to use.
Would it be possible to add that function? I think in hafs the output is something like "departurePlatform" 
thx

**Comment by derhuerst** [comment]
Yes, the platform is available via `departure.platform`/`departure.scheduledPlatform`.

**Comment by unclesam87** [comment]
but there isnt a function to display them yet, isnt their?

**Comment by derhuerst** [comment]
> but there isnt a function to display them yet, isnt their?

I don't know because I don't use `MMM-PublicTransportHafas`. I just maintain the underlying [`hafas-client` library](https://github.com/public-transport/hafas-client).

**Comment by KristjanESPERANTO** [comment]
I added platform support to [my fork](https://github.com/KristjanESPERANTO/MMM-PublicTransportHafas).
